spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -49,7 +49,7 @@
     end
   end
 
-  describe 'index' do
+  describe '#index' do
     specify 'pagination' do
       expect(Partner).to receive_message_chain(:all, :paginate).with(page: "2").and_return([].paginate)
```

Comment:
see my other comment in #308  on returning an array rather than `array.paginate` or what you have done for #index in the user controller. I think these are preferable.